suite-native/module-connect-device/src/screens/PinScreen.tsx

```diff
@@ -0,0 +1,28 @@
+import { useSelector } from 'react-redux';
+
+import { Image, VStack } from '@suite-native/atoms';
+import { DeviceModelInternal } from '@trezor/connect';
+import { selectDeviceModel } from '@suite-common/wallet-core';
+
+import { ConnectDeviceSreenView } from '../components/ConnectDeviceSreenView';
+import { PinForm } from '../components/PinForm';
+import { PinOnDevice, deviceImageMap } from '../components/PinOnDevice';
+
+export const PinScreen = () => {
+    const deviceModel = useSelector(selectDeviceModel);
+
+    if (!deviceModel) return null;
+
+    return (
+        <ConnectDeviceSreenView>
+            {deviceModel === DeviceModelInternal.T1B1 ? (
+                <VStack spacing="medium" alignItems="center" flex={1} marginTop="large">
```

Comment:
please, create and apply style for this

Reply:
I would rather keep it here as it was to save some conflicts resolutions, it's not a new change. We don't have some general rule when to use applyStyle, don't we?